core-api/src/main/java/com/optimizely/ab/odp/ODPManager.java

```diff
@@ -36,9 +36,18 @@ private ODPManager(@Nonnull ODPApiManager apiManager) {
     }
 
     private ODPManager(@Nonnull ODPSegmentManager segmentManager, @Nonnull ODPEventManager eventManager) {
-        this.segmentManager = segmentManager;
-        this.eventManager = eventManager;
-        this.eventManager.start();
+        this(segmentManager, eventManager, false);
+    }
+
+    private ODPManager(@Nonnull ODPSegmentManager segmentManager, @Nonnull ODPEventManager eventManager, boolean disable) {
+        if (!disable) {
```

Comment:
Java integrates ODP with builder pattern. ODPManager is expected to pass for initialization only when enabled.
I think we may not need this redundant "disable" control here.

enabled: 
```
    OptimizelyBuilder().withOdpManager(...).build()
```
disabled:
```
    OptimizelyBuilder().build()
```